public.openapi.json

```diff
@@ -277,7 +287,14 @@
         },
         "security": [
           {
-            "x-rh-identity": []
+            "x-rh-identity": [
+              "Type:System"
+            ]
+          },
+          {
+            "x-rh-idm-registration-token": [
+              "ipa-hcc"
```

Comment:
What is the purpose of the `ipa-hcc` value here?  I don't think it can really use this value to discriminate anything specifically about ipa-hcc (or other client) that would affect or augment the validity and authz checks performed on the token itself.

This is my only doubt/query.  Otherwise it's an ACK from me.

Reply:
I was worried that we could not distinguish missing scope from empty scope. Turns out Echo context stores data as `map[string]interface{}`, which gives different results for empty and missing. I'm going to remove the marker again.

```Go
package main

import "fmt"

type Map map[string]interface{}
var m Map

func main() {
	m = make(Map)
	m["filled"] = []string{"data"}
	m["empty"] = []string{}
	fmt.Println(m["filled"])
	fmt.Println(m["empty"])
	fmt.Println(m["missing"])
}
```

```
[data]
[]
<nil>
```